pkg/driver/camera/camera_windows.go

```diff
@@ -37,7 +37,7 @@ func init() {
 
 // Initialize finds and registers camera devices. This is part of an experimental API.
 func Initialize() {
-	C.CoInitializeEx(nil, C.COINIT_MULTITHREADED)
+	C.CoInitializeEx(nil, C.COINIT_APARTMENTTHREADED)
```

Comment:
What is this change?

Reply:
It's recommended to work with some UI framework, like Wails (https://wails.io/).

Reply:
I reverted it, and will open another PR.

Reply:
My English is bad, please forgive any mistakes.